packages/beeq/src/components/slider/bq-slider.tsx

```diff
@@ -45,6 +46,8 @@ export class BqSlider {
   @State() minValue: number;
   /** The `maxValue` state is only used when the slider type is `range`. */
   @State() maxValue: number;
+  @State() calculatedLeftThumbPosition: number;
+  @State() calculatedRightThumbPosition: number;
```

Comment:
Try to keep simple and meaningful state names:

```suggestion
  /** It hold the left position of the Thumb for the value or the minimum value (if the slider type is `range`) */
  @State() minThumbPosition: number;
  /** It hold the left position of the Thumb for the maximum value (if the slider type is `range`) */
  @State() maxThumbPosition: number;
```